source/docs/prismic/content-slices.md

```diff
@@ -0,0 +1,262 @@
+---
+id: content-slices
+title: Content Slices
+---
+
+[Content Slices](https://prismic.io/feature/dynamic-layout-content-components) allows developers to design autonomous and reusable UI elements that Content managers can use to build dynamic pages.
+
+Front-Commerce has first-class support of [Prismic Slices](https://prismic.io/docs/core-concepts/slices), and embraces the **"Don't Ship Pages, Ship a Page Builder"** vision. This page explains how developers can use the tool we provide to create content-driven dynamic pages.
+
+## How to use Slices?
+
+Integrating a Prismic Slice Zone and its Slices into your project is a 3 steps process:
+
+<!-- Workaround for https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one#table-of-contents -->
+<!-- no toc -->
+1. [expose data in GraphQL](#Expose-data-in-GraphQL)
+2. [create a Slice library](#Create-a-Slice-library) related to these GraphQL types. A library is composed of pairs of React Components and GraphQL fragments.
+3. [update your static pages to retrieve the content dynamically](#Retrieve-and-display-content) and use the SliceZone component of your library to display the data appropriately
+
+## Expose data in GraphQL
+
+In this example, we will add a Slice Zone with 3 types of Slices to an existing home page.
+
+First, **update your GraphQL Schema** to reflect the Slice Zone definition. A Slice Zone will be designed as a [GraphQL union type](https://graphql.org/learn/schema/#union-types).
+
+```diff
+type Homepage {
+  some_text: String
+  title: String
+  sales_date: Date
++  mainContent: [HomePageSlice]
+}
++
++ union HomePageSlice = Carousel | Push | FeaturedProducts
++
++ type FeaturedProducts {
++   sectionTitle: String
++   category: Category
++ }
++ type Carousel {
++   slides: [CarouselSlide]
++ }
++ type Push {
++   blocks: [PushBlock]
++ }
+```
+
+In your resolvers, load your content as usual [with the Prismic loader](/docs/prismic/expose-content.html#Methods-to-query-content).
+
+In addition to the `fieldTransformers` parameter (for fields), the loader methods accept a `supportedSlices` key. It allows to define to data format for Slices supported by the content type, when a Slice Zone is configured.
```

Comment:
```suggestion
In addition to the `fieldTransformers` parameter (for fields), the loader methods accept a `supportedSlices` key. It allows to define data format for Slices supported by the content type, when a Slice Zone is configured.
```

It a bit ambiguous imo

Reply:
I would even rephrase it with something like:

```suggestion
In addition to [the `fieldTransformers` parameter](/docs/prismic/expose-content.html#Field-Transformers), the loader methods accept a `supportedSlices` key. It allows to define to a data structure for the Slices available in the Custom Type.
```

Reply:
Thanks for the feedbacks. I struggled for a few minutes on this one and tried several things without being satisfied!